### PR TITLE
adding option to overwrite/update existing fields

### PR DESCRIPTION
**Pull request by chrisamiller**
Turns out the field I need for my use case (MQ0) is set by some, but not all callers. So some, but not all variants will have it.  Those without it will need it updated.  This PR should allow existing fields in the VCF to be updated.

### CODE REVIEW

File: vcf_annotation_tools/vcf_info_annotator.py [hunk 1]
```diff
@@ -32,13 +32,19 @@ def create_vcf_writer(args, vcf_reader):
...
+
     new_header = vcf_reader.header.copy()
-    od = OrderedDict([('ID', args.info_field), ('Number', '.'), ('Type', args.value_format), ('Description', args.description)])
-    if args.source:
-        od['Source'] = args.source
-    if args.version:
-        od['Version'] = args.version
-    new_header.add_info_line(od)
+
+    if args.w:
```
Comment (susannasiebert) [inline vcf_annotation_tools/vcf_info_annotator.py]: I don't think this handles the case where the `-w` flag is set but the desired `info_field` doesn't exist yet in the header. If that is the case we should create the header. I believe you can achieve this by adding `and args.info_field in new_header.info_ids()` to this if-clause.

A test case for this situation would probably also be good.

File: vcf_annotation_tools/vcf_info_annotator.py [hunk 2]
```diff
@@ -74,6 +80,9 @@ def define_parser():
         "-s", "--source",
         help="The string to put in the \"source\" section of the INFO header line - optional "
     )
+    parser.add_argument('-w', action='store_true',
```
Comment (susannasiebert) [inline vcf_annotation_tools/vcf_info_annotator.py]: What do you think of adding a long argument name here as well?

**Review by susannasiebert — CHANGES_REQUESTED** [review]

File: vcf_annotation_tools/vcf_info_annotator.py (hunk 2 above)
Comment (chrisamiller) [reply vcf_annotation_tools/vcf_info_annotator.py]: I tried that and it yelled at me :)  Let me try again

File: vcf_annotation_tools/vcf_info_annotator.py (hunk 1 above)
Comment (chrisamiller) [reply vcf_annotation_tools/vcf_info_annotator.py]: I think the right approach is to raise an exception if that happens - good call on finding that edge case and suggestion for the test. I'll add that

**Review by susannasiebert — APPROVED** [review]
